lib/ts/recipe/dashboard/api/multitenancy/listTenants.ts

```diff
@@ -38,16 +40,27 @@ export type Response = {
 export default async function listTenants(
     _: APIInterface,
     __: string,
-    ___: APIOptions,
+    options: APIOptions,
     userContext: any
 ): Promise<Response> {
+    const shouldIncludeUserCount = options.req.getKeyValueFromQuery("includeUserCount");
     let tenantsRes = await Multitenancy.listAllTenants(userContext);
     let finalTenants: TenantListTenantType[] = [];
 
     if (tenantsRes.status !== "OK") {
         return tenantsRes;
     }
 
+    // TODO: Add an API to core to get user count for all tenants in one go
```

Comment:
TODO

Reply:
This is no longer relevant since we have removed the user count from this API